cloudwash/entities/resources/discs.py

```diff
@@ -0,0 +1,51 @@
+from cloudwash.config import settings
+from cloudwash.entities.resources.base import DiscsCleanup
+from cloudwash.logger import logger
+from cloudwash.utils import dry_data
+
+
+class CleanDiscs(DiscsCleanup):
+    def __init__(self, client):
+        self.client = client
+        self._delete = []
+        self.list()
+
+    def _set_dry(self):
+        # VMsContainer = namedtuple('VMsCotainer', ['delete', 'stop', 'skip'])
+        # return VMsContainer(self._delete, self._stop, self._skip)
+        dry_data['DISCS']['delete'] = self._delete
```

Comment:
what is the `dry_data` here ? it's not a class or instance attribute ?

Reply:
I feel rather taking from the other module we should have it as class or instance variable ?

Reply:
@omkarkhatavkar Yeah making it a class is the next plan which I would work next and apart from this PR.